stable/distribution/UPGRADE_NOTES.md

```diff
@@ -59,3 +59,17 @@ This file describes special upgrade notes needed at specific versions
       4. Restore access to distribution
       5. Run `helm delete <OLD_RELEASE_NAME>` which will remove  old Distribution deployment and Helm release.
     * Distribution should now be ready to get back to normal operation
+
+## Upgrade from 4.x to 5.x and above (Chart Versions)
+
+* JFrog Distribution v2.x is only compatible with JFrog Artifactory v7.x and To upgrade, you must first install JFrog Artifactory 7.x
+* It is recommended to upgrade to the latest available chart versions. **Important** All the breaking changes should be resolved manually, see [changelog](https://github.com/jfrog/charts/blob/master/stable/distribution/CHANGELOG.md).
+
+* Upgrading to 7.x (chart version)
+  * Upgrade steps:
+    1. Delete the existing service and statefulset of distribution. kubectl delete statefulsets <OLD_RELEASE_NAME>-distribution, kubectl delete services <OLD_RELEASE_NAME>-distribution
```

Comment:
Use a code snippet for the `kubectl` commands

Reply:
Changed